extensions/jdbc/jdbc-oracle/runtime/src/main/java/io/quarkus/jdbc/oracle/runtime/graal/OracleAgroalConnectionConfigurer.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.jdbc.oracle.runtime.graal;
+
+import io.agroal.api.configuration.supplier.AgroalDataSourceConfigurationSupplier;
+import io.agroal.api.exceptionsorter.MySQLExceptionSorter;
+import io.quarkus.agroal.runtime.AgroalConnectionConfigurer;
+import io.quarkus.agroal.runtime.JdbcDriver;
+import io.quarkus.datasource.common.runtime.DatabaseKind;
+
+@JdbcDriver(DatabaseKind.MYSQL)
```

Comment:
Hum.... something looks wrong here?

Reply:
oops, this whole class shouldn't exist. It's not used either - deleding. Thanks!